File: leetcode/344.反转字符串/344.反转字符串.py
from typing import List


class Solution:
    def reverseString(self, s: List[str]) -> None:
        left = 0
        right = len(s) - 1
        while left < right:
            tmp = s[right]
            s[right] = s[left]
            s[left] = tmp
            left += 1
            right -= 1

File: leetcode/344.反转字符串/344.反转字符串.go
package main

func reverseString(s []byte) {
	left := 0
	right := len(s) - 1
	for left < right {
		tmp := s[left]
		s[left] = s[right]
		s[right] = tmp
		left += 1
		right -= 1
	}
}
